### ISSUE TITLE
Error "Could not resolve resource" in VS Code Extension

### ISSUE THREAD

**Issue by anthonywhite** [microsoft/vscode-azureresourcegroups#961]
Using Azure Resources VS Code extension v0.9.9, trying to expand a resource group in explorer. Get multiple of these errors:

```
info] Internal error: 'branchDataProvider.getResourceItem' threw an exception
	Could not resolve resource "/subscriptions/<subscription id>/resourceGroups/<resource group>/providers/Microsoft.Web/sites/<logic app name>"
		at CompatibleApplicationResourceBranchDataProvider.<anonymous> extension.bundle.js:2:1727792
		fulfilled extension.bundle.js:2:1726817
```

**Comment by alexweininger** [comment]
If you revert to past versions of Azure Resources, does the issue still reproduce? Are there any more logs in the Azure Resources log output channel?

**Comment by anthonywhite** [comment]
@alexweininger yes reproduces on e.g. version `0.9.7`. The above error repeats on every resource in the group, but there's no additional detail to add. Is there a way to get more verbose logging to track down the cause?

**Comment by alexweininger** [comment]
Ok, thanks for that information. For logs, you can check the Azure Resource Groups output window, and also the output windows of Logic Apps.

![Image](https://github.com/user-attachments/assets/e78c0960-4bc6-4da6-8041-63b1e3450b1d)
